packages/app-store/ee/routing-forms/components/SingleForm.tsx

```diff
@@ -262,3 +259,20 @@ export default function SingleForm({
     </Form>
   );
 }
+
+export default function SingleFormWrapper({ form: _form, ...props }: SingleFormComponentProps) {
+  const { data: form, isLoading } = trpc.useQuery(["viewer.app_routing_forms.formQuery", { id: _form.id }], {
```

Comment:
Because we are using tRPC now, it is important that useForm is called in a child component which is rendered only when `form` is available. THis is to avoid initialization of useForm with defaultValues:undefined